test/configs/zombie_tanssi_relay_starlight.json

```
@@ -0,0 +1,180 @@
+{
+    "settings": {
+        "timeout": 1000,
+        "provider": "native"
+    },
+    "relaychain": {
+        "chain_spec_path": "specs/tanssi-starlight-relay.json",
```

```suggestion
        "chain_spec_path": "specs/tanssi-relay-starlight.json",
```

Mismatch with build-spec script